com.inklestudios.Inky.appdata.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.inklestudios.Inky.desktop</id>
+  <name>Inky</name>
+  <project_license>MIT</project_license>
+  <developer_name>Inkle Studios</developer_name>
+  <summary>Write interactive narrative in inkle's markup language</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://www.inklestudios.com/ink/</url>
+  <url type="help">https://github.com/inkle/inky/issues</url>
+  <url type="bugtracker">https://github.com/inkle/inky/issues</url>
+  <description>
+    <p>Inky is a editor for ink, inkle's markup language for writing
+    interactive narrative in games, as used in 80 Days.</p>
+    <p>It's an IDE (integrated development environment), because it
+    gives you a single app that lets you play in the editor as you
+    write, and fix any bugs in your code.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://www.inklestudios.com/ink/img/inky-screenshot@2x.png</image>
+    </screenshot>
+  </screenshots>
+  <categories>
+    <category>IDE</category>
+    <category>Development</category>
+  </categories>
```

Comment:
```suggestion
```
The categories are merged from the desktop file, you don't have to repeat that in the metainfo file